sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
-});
+```javascript
+const myEventHandler = async (event) => {
+  // your code here
+};
+const myErrorHandler = (error) => {
+  console.log(error);
+};
+client.receive("partitionId", myEventHandler, myErrorHandler);
 ```
 
-## Example 3 - Send an event with partition key.
+### Get an event hub runtime info
 
-Send an event with a given "partition-key" which is then hashed to a partition ID (so all events with the same key will go to the same ID, but load is balanced between partitions). 
+Once you have created an instance of an `EventHubClient` class, get an event hub run time information 
+using the [getHubRuntimeInformation](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#gethubruntimeinformation--) function.
 
-```js
-const { EventHubClient, EventPosition } = require('@azure/event-hubs');
+```javascript
+const client = EventHubClient.createFromConnectionString("connectionString");
+await client.getHubRuntimeInformation();  
+```
 
-const client = EventHubClient.createFromConnectionString(process.env["EVENTHUB_CONNECTION_STRING"], process.env["EVENTHUB_NAME"]);
+### Get info about the specified partition
 
-async function main() {
-  // NOTE: For receiving events from Azure Stream Analytics, please send Events to an EventHub where the body is a JSON object.
-  // const eventData = { body: { "message": "Hello World" }, partitionKey: "pk12345"};
-  const eventData = { body: "Hello World", partitionKey: "pk12345"};
-  const delivery = await client.send(eventData);
-  console.log("message sent successfully.");
-}
-
-main().catch((err) => {
-  console.log(err);
-});
+Once you have created an instance of an `EventHubClient` class, get information about the specified partition
+using the [getPartitionInformation](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubclient?view=azure-node-latest#send-eventdata--string---number-) function.
+
+```javascript
+const client = EventHubClient.createFromConnectionString("connectionString");
+await client.getPartitionInformation("partitionId);  
 ```
 
-## Example 4 - Send an event to a specific partition id.
+**Note:** For scalable and efficient receiving, please take a look at [azure-event-processor-host](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-processor-host). The Event Processor host, internally uses the streaming receiver to receive events.
 
-Send an event to a specific partition ID if needed. If not specified then EventHub will store the events in the partition in a round-robin pattern.
+## IDE ##
+This sdk has been developed in [TypeScript](https://typescriptlang.org) and has good source code documentation. It is highly recommended to use [vscode](https://code.visualstudio.com) 
+or any other IDE that provides better intellisense and exposes the full power of source code documentation.
 
-```js
-const { EventHubClient, EventPosition } = require('@azure/event-hubs');
+### Enable logs
```

Comment:
Use the same heading as the Service Bus README here, I think it was Troubleshooting or something like that